gulpfile.js

```diff
@@ -64,10 +59,17 @@ gulp.task('set-environment-config', function (done) {
       });
     }
   });
-});
+};
+gulp.task('set-environment-config', set_environment_config);
+gulp.task('set-env', set_environment_config); // alias
```

Comment:
We should just choose one task name. I'm fine with less verbose naming if you prefer that.
The style of defining the function than calling the gulp.task method is fine but not used elsewhere in the file. If we don't need the alias task name.

Reply:
Then i would rename the main function and remove the alias.
The style i choose exactly after reading your mentioned link, because of cleaner console output.